docs/docs/guides/integrate/identity-providers/ldap.mdx

```diff
@@ -3,37 +3,33 @@ title: Configure LDAP as Identity Provider
 sidebar_label: LDAP
 ---
 
-import GeneralConfigDescription from './_general_config_description.mdx';
 import Intro from './_intro.mdx';
+import HowLDAPIDPWorks from './_how_ldap_idp_works.mdx'
 import CustomLoginPolicy from './_custom_login_policy.mdx';
+import IDPsOverview from './_idps_overview.mdx';
+import GeneralConfigDescription from './_general_config_description.mdx';
 import Activate from './_activate.mdx';
 import TestSetup from './_test_setup.mdx';
 
 <Intro provider="an LDAP server"/>
```

Comment:
I think we should also include the warning here. Because if it is ldap the passwords will be sent in clear text and this should be used with caution.

Reply:
you mean this warning https://github.com/zitadel/zitadel/pull/5671
right?

Reply:
I added this commit https://github.com/zitadel/zitadel/pull/5677/commits/d0d7cc6cc2a6feff7ba128d3148b7b3ecbd6116a